Source/Tools/LinuxEmulation/LinuxSyscalls/SignalDelegator.cpp

```diff
@@ -1405,7 +1404,7 @@ void SignalDelegator::HandleGuestSignal(FEXCore::Core::InternalThreadState* Thre
 #else
         // X86 should always be doing a refcount compare and branch since we can't guarantee instruction size.
```

Comment:
Guessing this comment needs an update ?

Reply:
Nah, x86 implementation still does a refcount compare and branch. That's over in SignalScopeGuards.h